pkg/config/analysis/incluster/controller.go

```diff
@@ -51,8 +51,20 @@ func NewController(stop <-chan struct{}, rwConfigStore model.ConfigStoreControll
 		"", resource.Namespace(namespace), func(name collection.Name) {}, true)
 	ia.AddSource(rwConfigStore)
 	// Filter out configs watched by rwConfigStore so we don't watch multiple times
+	schemas := collections.All.
+		Remove(rwConfigStore.Schemas().All()...)
+
+	// remove cluster-scoped resources if working in multi-tenant mode (with member roll controller)
+	if kubeClient.GetMemberRoll() != nil {
```

Comment:
When working on cluster-scoped mode, will this return nil?

Reply:
Yes. We don't set the member roll in cluster-scoped mode.